lib/yardstick/v2_client/booking_request.rb

```diff
@@ -24,26 +24,30 @@ def self.process_response(resp, extras = {})
         attrs
       end
 
-      def schedule
-        response = put(instance_action_uri(:schedule), body: { token: token, booking_request: {
-          local_start_at: local_start_at.utc.iso8601,
-          local_end_at: local_end_at.utc.iso8601
-        } })
+      def schedule(start_at)
+        response = put(instance_action_uri(:schedule), body: {
+          token: token,
+          booking_request: {
+            local_start_at: start_at.utc.iso8601
+          }
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.